components.tsx

```diff
@@ -192,11 +193,57 @@ export function PostPage({ post, state }: PostPageProps) {
         />
       </article>
 
+      {state.section || <Section title={post.title} href={href} />}
+
       {state.footer || <Footer author={state.author} />}
     </div>
   );
 }
 
+type Link = {
+  icon: VNode;
+  uri: string;
+};
+
+const sharingLinks: Record<string, Link> = {
+  "Twitter": {
+    icon: <IconTwitter />,
+    uri: "https://twitter.com/intent/tweet?text=",
+  },
+  "WhatsApp": {
+    icon: <IconWhatsApp />,
+    uri: "https://api.whatsapp.com/send?text=",
+  },
+  "Telegram": {
+    icon: <IconTelegram />,
+    uri: "https://telegram.me/share/url?url=",
+  },
+};
```

Comment:
I'm fine with adding a generic "section" that you can embed any JSX into (using configuration object to `blog()` function). However having hardcoded different sharing links seems too specific. Please rework this bit to provide a generic container without any content by default.